sympy/solvers/ode/single.py

```diff
@@ -59,8 +59,12 @@ def __init__(self, eq, func, sym):
         self.eq = eq
         self.func = func
         self.sym = sym
-        self.order = ode_order(eq, self.func)
 
+    @property
+    def order(self):
+        if self._order is None:
```

Comment:
Did you test this? I expect it to raise AttributeError because `_order` has not been initialised to None.

You should add a test for this (and run the tests before committing).

Reply:
sorry i just rushed,i will take care now.